waiter/test/waiter/core_test.clj

```diff
@@ -500,6 +509,48 @@
           (is (= http-500-internal-server-error status))
           (is (= expected-json-response-headers headers)))))
 
+    (testing "service-handler:delete-multiple-router-response-agree"
+      (reset! delete-service-result-atom {:result :deleted, :service-id service-id})
+      (reset! make-inter-router-requests-async-fn-atom (constantly {"r1" (async/go {:body (json/write-str {"exists?" false})})
+                                                                   "r2" (async/go {:body (json/write-str {"exists?" false})})}))
```

Comment:
We should include a router that does not include `exists?` in the response json.